hubblestack/daemon.py

```diff
@@ -690,6 +708,24 @@ def __init__(self, message, levelname, asctime, name):
         handler = hubblestack.splunklogging.SplunkHandler()
         handler.emit(MockRecord(__grains__, 'INFO', time.asctime(), 'hubblestack.grains_report'))
 
+def emit_to_syslog(grains_to_emit):
+    '''
+    Emit grains and their values to syslog
+    '''
+    try:
+        # Avoid a syslog line to be longer than 1024 characters
+        # Build syslog message
+        syslog_list = []
+        syslog_list.append('hubble_syslog_message:')
+        for grain in grains_to_emit:
+            if grain in __grains__:
+                syslog_list.append('{0}={1}'.format(grain, __grains__[grain]))
+        syslog_message = ' '.join(syslog_list)
```

Comment:
Some grains have spaces in them, which may make this hard to parse on the other end. I'd recommend using a different separator -- either a tab or a newline, probably.

Reply:
In syslog we can't do newlines and tabs may be not be nice and tricky to parse or look at on the host syslog log file. On mine if I replace the space with tab it will look like this:
```
hubble.py[14194]: hubble_syslog_message:#011system_uuid=92C3F82D-CDD1-4170-904A-AC6459A94287#011hubble_uuid=954d118d-5cb2-4023-9fb3-6a89c330fa17#011session_uuid=...
```

Reply:
After chatting with @fossam on slack we're going to leave this as is for now.